trusted-firmware-m/platform/ext/target/nordic_nrf/common/nrf5340/partition/flash_layout.h

```diff
@@ -148,6 +150,11 @@
                                          FLASH_ITS_AREA_SIZE)
 #define FLASH_NV_COUNTERS_AREA_SIZE     (FLASH_AREA_IMAGE_SECTOR_SIZE)
 
+/* Non-secure storage region */
+#define NRF_FLASH_NS_STORAGE_AREA_OFFSET    (FLASH_TOTAL_SIZE - \
```

Comment:
What is the motivation for placing this at the end of flash instead of directly behind the `NV_COUNTER` area? I assume there is some motivation since the placement of this area differs from all the others.

Reply:
Because the device tree for our chips places the storage partition at the end of the flash.